examples/bpk-component-chip-group/examples.tsx

```diff
@@ -189,6 +189,17 @@ export const OnImageChipGroup = () => {
   );
 };
 
+export const BpkChipGroupWithLabel = () => (
+     <div className={getClassName('bpk-chip-group-examples__with-label')}>
+       <BpkChipGroupState
+         type={CHIP_GROUP_TYPES.rail}
+         chips={chips}
+         chipStyle={CHIP_TYPES.default}
+         label='Filter'
+         ariaLabel="Select cities to filter by"
+       />
+     </div>
+   );
```

Comment:
Is it worth having one with the sticky chip on also?

Reply:
They are meant to be mutually exclusive. First we could tackle making that so and then add an example as apt depending on the expected behaviour.